client/webserver/locales/en-us.go

```diff
@@ -274,4 +274,11 @@ var EnUS = map[string]string{
 	"Disabled":                    "Disabled",
 	"txfee_not_available":         "Transaction fee currently unavailable",
 	"fiat_exchange_rate_msg":      "Sources may provide fiat exchange rates for different subsets of assets. You should select all sources that are acceptable to get fiat exchange rates for the most assets. Assets with fiat rates from multiple sources will use the average from all of those sources. Note: dcrdate provides fiat exchange rates for only BTC and DCR.",
+	"delete_archived_records":     "Delete Archived Records",
+	"date_time":                   "Date and Time",
+	"archived_date_time_tooltip":  "Archived orders and matches created before your specified and date and time will be deleted from the database. Note: If an invalid date or time is provided, all archived records till date will be deleted.",
```

Comment:
The text should be "all archived records will be deleted" instead of "all archived records **till date** will be deleted." However, I think the user should be able to select to either delete all the records, or if the date is invalid, then there should be an error message. I don't think an invalid date should delete everything.

Reply:
Agree it is unexpected to have that behavior when leaving the date field of the UI empty.  I only did that accidentally.  A "Specify date of latest records to keep" checkbox could work (or reverse wording).  By default, unchecked, and no data selection/entry box shown, and `null` sent to backend; if checked, show the date entry, and reject submission if no valid date entered.

I'm a little less concerned with summarizing after the purge, but ideally it would report on the location etc.

Reply:
Okay.

Reply:
> Right now there's no feedback after doing the deletion.

@chappjc,  do you think we need to modify the existing API to return `no of records deleted`?

Reply:
The mod to the db api is minor, and since we added a core method it's ok to change that.
Don't spend a lot of time on it, but it's worth a shot at showing some kind of result.

Reply:
<img width="339" alt="Screenshot 2022-10-29 at 9 53 28 AM" src="https://user-images.githubusercontent.com/57448127/198832194-349847f2-c50f-4e4e-b962-987027d3709b.png">

Reply:
My recent commit at https://github.com/decred/dcrdex/pull/1916/commits/d64637e9bf22dda58c09350f1e0a62f044e55b8a also fixed an issue where files are created even if no record was deleted.